MAINTAINERS.rst

```diff
@@ -88,12 +88,15 @@ Building and uploading the release wheels
 You need the zephyr-project PyPI credentials for the 'twine upload' command. ::
 
   git clean -ffdx
-  python3 setup.py sdist bdist_wheel
+  pip3 install --upgrade build twine
```

Comment:
I'm still not a fan of using the "random version of the day" but these instructions seem to work now and this file is only for MAINTAINERS who are supposed to know what they're doing.